app/controllers/webhooks/base_controller.rb

```diff
@@ -0,0 +1,16 @@
+class Webhooks::BaseController < ApplicationController
+  rescue_from ApiErrors::StandardError, with: :stripe_error
```

Comment:
Hmm If I create a Stripe Specific Controller on top of base controllers, what should i name the error here?   
```   
def validate_event_type
    if !StripeWebhookEvents::VALUES.include?(params[:type])
      raise ApiErrors::StandardError
    end
  end
```